lib/databases/mongodb.js

```diff
@@ -41,18 +41,21 @@ var MongoSessionStore = function (options) {
   _.defaults(options, defaults);
 
   var defaultOpt = {
-    ssl: false
+    // ssl is by default false in node-mongodb-native driver v1.4.x / v2.2 and above
+    // ssl: false
   };
 
   options.options = options.options || {};
 
   if (isNew) {
-    defaultOpt.autoReconnect = false;
+    // autoReconnect needs to be true to reconnect on failure, by default it is true in node-mongodb-native driver v2.2 and above
```

Comment:
it is intentionally set to false, 
cloud-native environment, if there is no connection to the db, the process is killed...